src/utils/width.css

```diff
@@ -0,0 +1,107 @@
+.u-w-full {
+  width: 100%;
+}
+
+.u-w-1\/2 {
```

Comment:
I think the naming could use some work, as the name doesn't reflect this is different to the grid `.u-1/2` class. I'd say there are two options:

- Rename this one to be separate to the other one (e.g. this one is "flex")
- Rename both so that both are more clear

Reply:
In that case I would probably want to rename the CSS grid version. Since that will be used way less (Only on the outer container I believe, right?)

Reply:
Also, this only sets the actual width. I'll look at the other one. I'll probably rename that to: `u-grid-1/2` etc

Reply:
@jvanbaarsen Myself and @wesoudshoorn were discussing using CSS grid for all the block level container elements, but you can choose whichever way you feel more comfortable with. 👍

Reply:
@xadamy Is CSS grid also useable within a card for example? I had a call with Wes today, and he believed the CSS Grid setup is not useable for that.

Reply:
That's probably the level you'd want to switch to flex, indeed. By block level elements, I mean the biggest container elements on the page e.g. the card itself, but not the contents of the card.

Reply:
Ah yeah, in that case I probably prefer this approach. I’ll do some testing later this week